src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PortableInfoboxObject.java

```diff
@@ -0,0 +1,245 @@
+    Assertion.assertEquals(isElementOnPage(title), true);
+    return this;
+  }
+
+  public PortableInfoboxObject isTitleAboveImageVisible() {
+    wait.forElementVisible(titleSmallImage);
+    Assertion.assertEquals(isElementOnPage(titleSmallImage), true);
+    return this;
+  }
+
+  public PortableInfoboxObject isImageInTitleNotVisible() {
+    Assertion.assertEquals(isElementVisible(imageInTitle), false);
+    return this;
+  }
+
+  public PortableInfoboxObject isInfoboxCollapsed() {
+    Assertion.assertEquals(isElementOnPage(infoboxIsCollapsed), true);
+    return this;
+  }
+
+  public PortableInfoboxObject isInfoboxExpanded() {
+    Assertion.assertEquals(infoboxWrapper.getAttribute("style"), "height: auto;");
+    return this;
+  }
+
+  public PortableInfoboxObject isImageInTabberVisible() {
+    Assertion.assertEquals(isElementVisible(imageInTabber), true);
+    return this;
+  }
+
+  public PortableInfoboxObject isImageCaptionInTabberVisible() {
+    Assertion.assertEquals(isElementVisible(captionInTabber), true);
+    return this;
+  }
+
+  public PortableInfoboxObject isVideoVisible() {
+    Assertion.assertEquals(isElementVisible(video), true);
+    return this;
+  }
+
+  public PortableInfoboxObject isVideoCaptionVisible() {
+    Assertion.assertEquals(isElementVisible(videoCaption), true);
+    return this;
+  }
+
+  public PortableInfoboxObject isHeroImageCentered() {
+    Assertion.assertEquals(images.get(0).getCssValue("text-align"), "center");
+    return this;
+  }
+
+  public PortableInfoboxObject areBoldElementsPresented() {
+    Assertion.assertFalse(boldElements.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject areItalicElementsPresented() {
+    Assertion.assertFalse(italicElements.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject areUnorderedListsVisible() {
+    Assertion.assertFalse(unorderedLists.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject areOrderedListsVisible() {
+    Assertion.assertFalse(orderedLists.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject areHeadersVisible() {
+    Assertion.assertFalse(headers.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject verifyDataItemsVisibility() {
+    Assertion.assertFalse(dataLabels.isEmpty());
+    Assertion.assertFalse(dataValues.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject verifyLinksVisibility() {
+    Assertion.assertFalse(externalLinks.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject verifyReferencesVisibility() {
+    Assertion.assertFalse(references.isEmpty());
+    return this;
+  }
+
+  public PortableInfoboxObject verifyExternalLinkNameAndURL(String name, String URL) {
+    Assertion.assertStringContains(URL, name);
+    return this;
+  }
+
+  public PortableInfoboxObject compareListsAndDataValuesMargin() {
+    Assertion.assertEquals(
+        unorderedLists.get(0).getCssValue("margin"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "margin" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)